src/stream/src/cache/lru_manager.rs

```diff
@@ -166,6 +173,13 @@ impl LruManager {
                 watermark_time_ms += self.barrier_interval_ms as u64 * step;
             }
 
+            self.metrics
+                .lru_current_watermark_time_ms
+                .set(watermark_time_ms as i64);
+            self.metrics.lru_physical_now_ms.set(physical_now as i64);
+            self.metrics.lru_watermark_step.set(step as i64);
+            self.metrics.lru_runtime_loop_count.inc();
+
```

Comment:
Can we also add a metric for `jemalloc_allocated_mib`? Because sometimes the memory shown in Grafana is much more than specified threshold `total_memory_available_bytes`

Reply:
Sure